crates/typst-library/src/math/attach.rs

```diff
@@ -129,14 +129,12 @@ pub struct LimitsElem {
     pub inline: bool,
 }
 
-/// Stretches a glyph.
+/// 字形を伸縮します。
 ///
-/// This function can also be used to automatically stretch the base of an
-/// attachment, so that it fits the top and bottom attachments.
+/// この関数は、上部及び下部アタッチメントがフィットするように、自動的にアタッチメントのベースを伸縮させることにも使えます。
 ///
-/// Note that only some glyphs can be stretched, and which ones can depend on
-/// the math font being used. However, most math fonts are the same in this
-/// regard.
+/// 伸縮可能な字形は限られており、どの字形が伸縮可能かは使用する数式フォントに依存することに注意してください。
+/// しかしながら、この点に関してほとんどの数式フォントに違いはありません。
```

Comment:
現状でも問題はないです。参考になれば幸いです。採否はお任せいたします。

「しかし」→「ただし」（全否定というよりは一部否定な感じを出すため）
「数式フォントに」→「数式フォントで」
「違いはない」→「共通している」
としました。

```suggestion
/// ただし、この点については多くの数式フォントで共通しています。
```

Reply:
ありがとうございます。
「違いはない」→「共通している」は個人的に意味が取りづらかったため、それ以外を採用させてもらいました。